azext_iot/_help.py

```diff
@@ -518,6 +518,57 @@
     short-summary: Upload a local file as a device to a pre-configured blob storage container.
 """
 
+helps['iot device get-parent'] = """
+    type: command
+    short-summary: Get the parent device of the specified device.
+    examples:
+    - name: Get the parent device of the specified device .
+      text: >
+        az iot device get-parent -d [Non_Edge_Device_Id] -n [IoTHub_Name]
+"""
+
+helps['iot device children'] = """
+    type: group
+    short-summary: Manage Edge offline.
+"""
+
+helps['iot device children add'] = """
```

Comment:
I am not sure if the spec got updated or not, I remember we need to implement children SET command as well? I could be wrong, can you double check with Venkat?

Reply:
Not required, as remove-all and add will be "set" functionality